frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -575,6 +576,40 @@ class TracingApi {
     return result;
   }
 
+  measureTreeLength(tree: Tree): number {
```

Comment:
As there is no api function that returns a specific tree (only `getAllTrees`), it would be preferable if this function would work with a tree id instead of a tree. Or you could add an api function that returns a tree with a specific id, not sure about the best solution.

Reply:
Good point! I changed it to use ids only :+1: